docs/API.md

```diff
@@ -82,6 +82,7 @@ interface ErrorOverlayOptions {
   sockHost?: string;
   sockPath?: string;
   sockPort?: number;
+  sockProtocol?: 'http' | 'https';
```

Comment:
I did a quick search on GitHub and I think `ws` and `wss` protocols are getting more traction - we should also allow them here (please update all other places related to this enum too!)